src/components/select/SingleSelectConnected.tsx

```diff
@@ -13,6 +13,7 @@ export interface ISingleSelectOwnProps extends ISelectProps {
     placeholder?: string;
     toggleClasses?: string;
     onSelectOptionCallback?: (option: string) => void;
+    disabled?: boolean;

```

Comment:
buttonProps: React.HTMLProps<HTMLButtonElement>

Reply:
I would even go further and remove it. We should go the same way as #644 by using the `React.Component<ISingleSelectProps & React.HTMLProps<HTMLButtonElement>` in combinaison with `ts-transformer-keys` and the `{..._.omit(this.props, keys<ISingleSelectProps>()}`